.github/workflows/deploy.yml

```
@@ -0,0 +1,126 @@
+              ${{ steps.login-ecr.outputs.registry }}/${{ secrets.AWS_ECR_REPOSITORY }}:${{ github.sha }}
+
+            # 6) Health check (최대 1분 대기)
+            echo "Waiting for application to start..."
+            for i in {1..30}; do
+              if curl -fsS http://localhost/actuator/health | grep -q '"status":"UP"'; then
+                echo "Health check successful!";
+                exit 0;
+              fi;
+              sleep 2;
+            done
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)